include/fbgemm/FbgemmConvert.h

```diff
@@ -12,7 +12,7 @@
 
 namespace fbgemm {
 
-typedef uint16_t bfloat16;
+using bfloat16 = int16_t;
```

Comment:
Define this in https://github.com/pytorch/FBGEMM/blob/5209a671f31f2bb2ec77ba581d38336fa820faa2/include/fbgemm/Types.h#L15 ?

Reply:
Hi, @jgong5. This is because uint16_t  is float16, and I need to use int16_t to distinguish them with the template.
@jianyuh, I moved  `using bfloat16 = std::int16_t;` in Types.h and add two inline functions(bf16_2_float, float_2_bfloat16) there to avoid introducing too many overload functions to pass the compile.

Reply:
> Hi, @jgong5. This is because uint16_t is float16, and I need to use int16_t to distinguish them with the template.

Can you note this in the code?

Reply:
Will do.